src/Data/String/Unsafe.agda

```diff
@@ -21,3 +24,19 @@ toList∘fromList s = trustMe
 
 fromList∘toList : ∀ s → fromList (toList s) ≡ s
 fromList∘toList s = trustMe
+
+toList-++ : ∀ s t → toList (s ++ t) ≡ toList s List.++ toList t
+toList-++ s t = trustMe
```

Comment:
Any thoughts on what we need to add to 2.6.2 in order to make these safe?

Reply:
It would make sense to add axioms specifying these BUILTIN functions.
These types are pretty annoying to write in internal syntax though! :D